libcodechecker/util.py

```diff
@@ -373,3 +375,28 @@ def get_line(file_name, line_no):
             return ''
     except IOError:
         return ''
+
+
+class TemporaryDirectory:
+    def __init__(self, suffix='', prefix='tmp', dir=None):
+        self._closed = False
```

Comment:
Why do we need the `self._closed` attribute?

Reply:
To prevent if the user calls explicitly the cleanup method several times.